### PR TITLE
Engineering Sprite Cleanup

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Yet another entry in my cleanup of that hellish uniforms.dmi. This one's by far the smallest as no one sprites anything for engineering.

No notable changes in this one, entirely just re-pathing and the re-linking. That's how untouched engineering is down-stream here.

## How This Contributes To The Skyrat Roleplay Experience
Cleaner code/icon organization

## Changelog

:cl:
imageadd: Engineering's uniform/digi icons have all been moved into clothing/under/engineering.dmi files in a continued organization effort. Please report any Engineering Uniform icon issues to @Orion_The_Fox#1915 if they snuck in.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
map errors arent me i swear

**Comment by theselfish** [comment]
Aren't there engineering trousers that someone added?

File: modular_skyrat/master_files/code/modules/clothing/under/jobs/engineering.dm [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/under/engineering_digi.dmi' //Anything that was in the engineering.dmi, should be in the engineering_digi.dmi
+
+/obj/item/clothing/under/rank/engineering/engineer/skyrat
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/engineering.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/engineering.dmi'
+
+//Add a /obj/item/clothing/under/rank/engineering/chief_engineer/skyrat or /obj/item/clothing/under/rank/engineering/atmospheric_technician/skyrat if you add uniforms for either
+
+//////////////////////////////////////
+////////////// ENGINEER //////////////
```
Comment (tf-4) [inline modular_skyrat/master_files/code/modules/clothing/under/jobs/engineering.dm:11]:
```suggestion
/*
ENGINEER
*/
```

could lead to autodoc weirdness

**Comment by ghost** [comment]
> Aren't there engineering trousers that someone added?

we dont talk about those

plus they need to be tweaked so I'm not including them in any of these PRs

File: modular_skyrat/master_files/code/modules/clothing/under/jobs/engineering.dm (hunk 1 above)
Comment (ghost) [reply modular_skyrat/master_files/code/modules/clothing/under/jobs/engineering.dm:11]: woe is I
I've not seen it cause issues but better safe than sorry; I'll hit all the other ones I added in the other cleanup PRs too

Comment (ghost) [reply modular_skyrat/master_files/code/modules/clothing/under/jobs/engineering.dm:11]: wait, no, it doesnt cause issues, TG uses it a lot already for similar sorting (see: packs.dm)

**Pull request merged**